### ISSUE TITLE
Just not working

### ISSUE THREAD

**Issue by rorc** [se-panfilov/mini-toastr#1]
http://joxi.ru/82Q7PBEHa0Gard

Even if i install babel and babel-runtime i catch this error: http://joxi.ru/4AkOKgaF8aa0Aq

My vue version is 2.0.3.

**Comment by se-panfilov** [comment]
@rorc Hey, first of all thanks for report.

Honestly this project is in age of a few weeks atm. It's super young. So, the bugs can be occurred.

The second thing is that **i didn't tested it with vue 2.x.x**. 

In theory it have to work, but y'know, have to be tested first to make sure.

Also.

Which version of mini-toastr do you use? Latest?

**Comment by rorc** [comment]
@se-panfilov Hey) I am using last version `^0.3.9`
Now all is working, thanks!

**Comment by rorc** [comment]
@se-panfilov Btw, whats difference between mini-toastr and vue-notifications?

**Comment by se-panfilov** [comment]
@rorc 

`vue-notifications` and `mini-toastr` it's a completely different things.

tldr:
 - `vue-notifications`  - it's just an interface for work with notifications. It've got to be used with any other library that implement notifications as is. In my examples you can wire-up it with `mini-toastr` (or any other library), and after that `vue-notifications` would send all the messages to that lib;
 - `mini-toastr` - it's a small library that can display notifications (ui elements), like `.suceess()`, `errorr()`, etc. It's can be used in any kind of projects, not only vuejs.


======

More details:

`vue-notifications` it's a kind of an interface that allow you to setup notifications in declaration style, like

```JS
export default {
    name: 'DemoView',
    data () {
      //...
    },
    methods: {
      //...
    },
    notifications: { // This isa kind of config. When you will call `showLoinError` this setup would be used
      showLoinError: {
        title: 'Login Failed',
        message: 'Failed to authenticate',
        type: 'error' //Default: 'info'
      }
    }  }
```

after that you're will be allowed to work like `this.showLoinError()`

But by itself `vueNotifications` send messages only to console (like `console.log()`).

The idea here is to allow you to replace any kind of notification library (like `mini-toastr`) easy in future. 

---

`mini-toastr` - it's a kind of simple non-blocking notifications. can be used without any frameworks at all. 

Take a look at this

```JS
function toast ({title, message, type, timeout, cb, consoleMessage}) {
  if (consoleMessage) console[type](consoleMessage) //Here we go!
  return miniToastr[type](message, title, timeout, cb)
}

const options = {
  success: toast,
  error: toast,
  info: toast,
  warn: toast
}

Vue.use(VueNotifications, options)
```

Here I order to `vue-notifications` to send all messages to `miniToastr` and how to call `mini-toastr`'s methods: `miniToastr[type](message, title, timeout, cb)`

**Comment by rorc** [comment]
@se-panfilov Ok, thanks for detailed answer =)
